generation/WinSDK/Partitions/Security.Cryptography/settings.rsp

```diff
@@ -15,5 +16,53 @@ _LM_OWF_PASSWORD
 <IncludeRoot>/um/i_cryptasn1tls.h
 <IncludeRoot>/um/infocard.h
 <IncludeRoot>/um/mssign.h
+<IncludeRoot>/cpdk/bcrypt_provider.h
```

Comment:
The recompile script I linked recursively moves everything under AdditionalHeaders to the `um` directory in the `<IncludeRoot>` folder, so I believe these `cpdk` paths need to be changed to `um` to resolve properly.

Reply:
Ah right, so a subsequent build after IDL recompile will fail. Will change that now.